lib/typescript/httpclient/model/Template.ts

```diff
@@ -5,5 +5,5 @@ export interface Template {
   name: string;
   sourceType: SourceType;
   content: string;
-  variables: any;
+  variables?: any;
```

Comment:
I'm removing this from the api as its introduction was premature (I planned to remove it from here as well but since we're at it we can do here)

Reply:
ahh ok, I thought it was only becoming optional! thanks for the update